26.Remove_Duplicates_from_Sorted_Array.py

```diff
@@ -0,0 +1,17 @@
+# https://leetcode.com/problems/remove-duplicates-from-sorted-array/
+#
+
+def removeduplicates(nums):
+    i = 0
+    l = len(nums)
+    while i <= l-2:
+        if nums[i] != nums[i+1]:
+            i += 1
+        else:
+            nums.remove(nums[i])
```

Comment:
here you modify the array but in the description it says that you should do that **in-place**
but the `remove` operation actually modifies it, so you just break the rule :)

the correct idea is to put all the duplicates to the back of the array, and store all unique values in the begging of your array.
the idea is the same as in 27th task